__tests__/app.test.js

```diff
@@ -52,38 +51,88 @@ describe("GET /api/articles/:article_id", () => {
    created_at
    votes
 `, () => {
-  return request(app)
-  .get("/api/articles/1")
-  .expect(200)
-  .then((res) => {
-    expect(res.body.article).toEqual({
-      article_id: 1,
-      title: "Living in the shadow of a great man",
-      topic: "mitch",
-      author: "butter_bridge",
-      body: "I find this existence challenging",
-      created_at: expect.any(String),
-      votes: 100,
-    });
-  })
-})
-}); 
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 100,
+        });
+      });
+  });
+});
 
 describe("GET /api/articles/invalid ID", () => {
```

Comment:
I'd like you to change this to something like GET error handling. The tests in here don't just cover invalid id, but there's also one other error handling test on line 128 that should be moved into here (or all of them moved under the happy path tests -either way is fine as long as it's consistent.)